### ISSUE TITLE
reduce file size

### ISSUE THREAD

**Issue by amanuense** [amanuense/DUWS-R#25]
pretty much remove dead code, remove unused files and try to reuse as much as possible

**Comment by ultranoobian** [comment]
The majority of the projects file size comes from the audio files included for radio chatter in (source/sounds/)

Which is 84% of the source folder!

**Comment by amanuense** [comment]
yup I have noticed that, we should get rid of those or if we use radio
chatter we could use smaller sources.

Oscar
